tga-tool/src/main/kotlin/org/plan/research/tga/tool/testspark/TestSparkCliTool.kt

```diff
@@ -118,7 +135,16 @@ class TestSparkCliTool(args: List<String>) : TestGenerationTool {
             )
             log.debug("Starting TestSpark with command: {}", processBuilder.command())
 
+            processBuilder.redirectErrorStream(true)
             process = processBuilder.start()!!
+
+            log.debug("Configure reader for the TestSpark process")
+
+            val reader = BufferedReader(InputStreamReader(process.inputStream))
+            var line = ""
+            while (reader.readLine()?.also { line = it } != null) {
+                println(line)
```

Comment:
1. If we need this output, I would say it is better to write it to log or  separate 'TestSpark' output file
2. If we don't need it --- we can just discard it https://docs.oracle.com/javase%2F9%2Fdocs%2Fapi%2F%2F/java/lang/ProcessBuilder.Redirect.html

Reply:
The TestSpark logs related to the test generation are printed in the console and also saved into `[project-output-dir]/[iteration]/[current-project-under-test]/generated-artifacts/test-generation.log` file by TestSpark itself (I can make TestSpark only write into the log file without printing if needed), other logs such as runtime errors are only printed in the console right now. 

For me these console logs were helpful to find TestSpark headless mode bugs if runtime exceptions are thrown due to which the tool aborts its execution.

I do not want to discard this output because if an error occurs I cannot spot it. So, I think it is better to log them in a separate file.

Reply:
Ok, that works. Thank you